src/Services/Flight/src/Flight/Flights/Features/CreatingFlight/V1/CreateFlightMongo.cs

```diff
@@ -14,7 +15,7 @@
 using MongoDB.Driver;
 using MongoDB.Driver.Linq;
 
```

Comment:
For mono commands, we shouldn't use value objects! Please revert it to normal case.

Reply:
understand